provision.yaml

```diff
@@ -0,0 +1,19 @@
+---
+default:
+  provisioner: docker
+  images: ['waffleimage/centos7']
+travis_ubuntu:
+  provisioner: docker
+  images: ['waffleimage/ubuntu14.04','waffleimage/ubuntu16.04','waffleimage/ubuntu18.04']
+travis_debian:
+  provisioner: docker
+  images: ['waffleimage/debian8']
+travis_centos7:
+  provisioner: docker
+  images: ['waffleimage/centos7']
+travis_oracle7:
+  provisioner: docker
+  images: ['waffleimage/oraclelinux7']
+release_checks:
+  provisioner: vmpooler
+  images: ['redhat-6-x86_64', 'redhat-7-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'oracle-6-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64',  'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64']
```

Comment:
helou @carabasdaniel ,

RedHat5, RedHat8, CentOS5, OracleLinux5,SLES are not included in provision file. Do we need to include them into testing? If not, please remove their support from metadata

Reply:
SLES is not included because of the dependent module puppetlabs-gcc that is not compatible with SLES. 

Litmus seems to have some issues applying idempotent manifest on CentOS 5, RedHat 5 and OracleLinux 5 and will need to be investigated to be able to add these after the conversion. 

I'll add RedHat 8 to the provision file for the release checks.